src/tests/gfx_gltf.edn

```diff
@@ -0,0 +1,34 @@
+(def timestep (/ 1.0 120.0))
+(def Root (Node))
```

Comment:
You can use `(defnode root)`. In most samples (in documentation), we use lowercase names but we haven't really defined a convention yet.